<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="group_1">
     <property name="title">
      <string>串口设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="s1__lb_1">
        <property name="text">
         <string>串口检测：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="s1__btn_port_check">
        <property name="text">
         <string>检测串口</string>
        </property>
        <property name="autoRepeatInterval">
         <number>100</number>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="s1__lb_2">
        <property name="text">
         <string>串口选择：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="s1__box_port_select"/>
      </item>
      <item>
       <widget class="QLabel" name="s1__lb_3">
        <property name="text">
         <string>波特率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="s1__box_baud_rate">
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>76800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>230400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>460800</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="s1__lb_state">
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="s1__btn_port_open">
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="s1__btn_port_close">
        <property name="text">
         <string>关闭串口</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_2">
     <property name="title">
      <string>取样器操作</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="2">
       <widget class="QLineEdit" name="s2__edit_init_pos">
        <property name="text">
         <string>22</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="s2__btn_next">
        <property name="text">
         <string>下一个位置</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="s2__btn_niddle_move">
        <property name="text">
         <string>针臂移动</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="s2__btn_init_pos">
        <property name="text">
         <string>初始位置纠正</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="s2__edit_niddle_move">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="s2__btn_forward">
        <property name="text">
         <string>上一个位置</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="s2__btn_turn_init">
        <property name="text">
         <string>转盘位置初始化</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="s2__btn_work">
        <property name="text">
         <string>开始工作</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QRadioButton" name="s2__rad_work">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>工作中</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_3">
     <property name="title">
      <string>取样器工作状态</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="8" column="1">
       <widget class="QLabel" name="s3__lb_2">
        <property name="text">
         <string>总位置：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="s3__box_turn_motor">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>转盘电机工作</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="s3__lb_1">
        <property name="text">
         <string>当前位置：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="s3__box_niddle_motor">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>针头电机工作</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="s3__box_turn_0">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>主转盘到达1号瓶位置</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="s3__edit_cur_pos">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>-1</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="s3__box_turn_init">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>转盘初始化</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="s3__edit_bottle_num">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>50</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QPushButton" name="s3__btn_update_pos">
        <property name="text">
         <string>更新位置信息</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="s3__box_niddle_limit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>针头限位</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
